dotCMS/src/main/java/com/dotcms/rest/api/v1/experiments/ExperimentForm.java

```diff
@@ -124,4 +135,39 @@ public ExperimentForm build() {
             return new ExperimentForm(this);
         }
     }
+
+    private void validateScheduling(final Scheduling scheduling) {
```

Comment:
is this the same? https://github.com/dotCMS/core/pull/22969/files#diff-5aa71567ff53ee229d25c2386cce3169c657f37a3e04cf27b75b3fd029537a7cR274

Reply:
Refactored to reuse